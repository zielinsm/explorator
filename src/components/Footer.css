.Footer {
    width: 100%;
    margin-top: 0.25em !important;
    margin-bottom: 0.25em !important;
    text-align: center;
    clear: both;
}

.Footer__disclaimer {
    font-size: 0.85em;
}
