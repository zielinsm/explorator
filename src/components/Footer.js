import React from "react";

import "./Footer.css";

function Footer() {
  return (
    <div className="Footer">
      <span className="Footer__disclaimer">
        The "Tor" wordmark is a registered trademark of the Tor Project.
      </span>
    </div>
  );
}

export default Footer;
