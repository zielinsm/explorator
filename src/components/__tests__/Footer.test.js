import React from "react";
import { shallow } from "enzyme";
import renderer from "react-test-renderer";

import Footer from "../Footer";

describe("Footer component:", () => {
  const component = <Footer />;

  it("renders without crashing", () => {
    const wrapper = shallow(component);
    expect(wrapper.find(".Footer")).toHaveLength(1);
  });

  it("snapshot matches", () => {
    const tree = renderer.create(component).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
