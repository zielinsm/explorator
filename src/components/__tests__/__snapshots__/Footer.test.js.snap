// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component: snapshot matches 1`] = `
<div
  className="Footer"
>
  <span
    className="Footer__disclaimer"
  >
    The "Tor" wordmark is a registered trademark of the Tor Project.
  </span>
</div>
`;
